docs/examples/neofs-ir/config.yml

```diff
@@ -0,0 +1,55 @@
+# Contfig file example
```

Comment:
This file will be duplicate of https://github.com/nspcc-dev/neofs-node/blob/master/config/example/node.yaml config example.